frontend/lib/common.js

```diff
@@ -6,10 +6,10 @@ import getEnvVars from '../environment';
 /**
  * Usage:
  * import { base } from "../lib/common.js"
- * BASE("Products")...
- **/
+ * base("Products")...
+ **/ 
 
-const { AIRTABLE_API_KEY, AIRTABLE_BASE_ID } = getEnvVars();
+const { AIRTABLE_API_KEY, AIRTABLE_BASE_ID, GOOGLE_MAPS_API_KEY } = getEnvVars();
```

Comment:
Do you want to remove this for this PR and leave a TODO comment instead? (Google maps key)